drone-webdriver/src/main/java/org/jboss/arquillian/drone/webdriver/utils/HttpClient.java

```diff
@@ -22,6 +27,7 @@ public Response get(String url, Map<String, String> headers) throws IOException
         try (CloseableHttpClient client = createHttpClient()) {
             final HttpGet request = new HttpGet(url);
             addHeaders(headers, request);
+            log.info("Sending request: " + request + " with headers: " + Arrays.asList(request.getAllHeaders()));
```

Comment:
Personally I am not interested to see if requests are sent. Only when something is not working I would be interested in seeing this information - so maybe debug with a simple way to enable it?